client/client.go

```diff
@@ -23,14 +46,82 @@ func NewFurtiveClient(ws *websocket.Conn) *FurtiveClient {
 
 func (fc *FurtiveClient) ReadMessages() {
 	for {
-		var msg *Message
+		var contents json.RawMessage
+		msg := &Message{
+			Contents: &contents,
+		}
```

Comment:
```suggestion
		var contents json.RawMessage
		msg := &Message{
			Contents: &contents,
		}
```
Is `var msg *Message` not enough?

Reply:
Not, I need that in lines 62 and 69

Reply:
Right - contents is of `interface{}` type.